packages/flutter/test/widgets/widget_inspector_structured_error_test.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+import 'dart:math';
+import 'dart:ui' as ui;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart' show DragStartBehavior;
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'widget_inspector_test.dart';
```

Comment:
don't import a test from another test. I would move TestWidgetInspectorService to a shared file used by multiple tests.

Reply:
I didn't see a specific area for utils so just added a _utils file. Not sure if that's what's intended.